setup.py

```diff
@@ -247,6 +247,23 @@ def initialize_options(self) -> None:
     gcs_dependencies
 )
 
+docs_dependencies = [
+    'sphinx>=3',  # used to generate and upload docs -
+                  # sphinx-autodoc-typehints requires 3 or better per
+                  # https://github.com/agronholm/sphinx-autodoc-typehints/pull/138
+    'sphinx-autodoc-typehints',  # used to handle type hints  # TODO will this be used?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker